apps/web/src/components/editor/tiptap.tsx

```diff
@@ -443,14 +444,42 @@ function TiptapWrapper(
     }
   }, [isHydrating]);
 
+  useEffect(() => {
+    if (!editorContainerRef.current) return;
+
+    const handleWheel = (e: WheelEvent) => {
+      if (e.ctrlKey) {
+        e.preventDefault();
+        const delta = e.deltaY;
+        if (delta > 0) {
+          editorConfig.zoom += EDITOR_ZOOM.STEP;
+        } else {
+          editorConfig.zoom -= EDITOR_ZOOM.STEP;
+        }
+        const zoom = Math.min(
+          EDITOR_ZOOM.MAX,
+          Math.max(EDITOR_ZOOM.MIN, editorConfig.zoom)
+        );
```

Comment:
You shouldn't directly change `editorConfig.zoom`. Also can we slow down the zoom in/out a bit on wheel scroll? It's too fast.